Curve-finance-suite/src/agents/ramp.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Finding,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+  HandleTransaction,
+} from 'forta-agent';
+
+export const RAMPSIGNATURE = 'Ramp(uint256,uint256,uint256,uint256)';
+
+export const createFinding = (alertID: string, address: string) => {
+  return Finding.fromObject({
+    name: 'Ramp',
+    description: 'Ramp Called',
```

Comment:
Consider making this Finding more informative. The `name` field is not descriptive, maybe you can add Curve somewhere and also add the fee values in the metadata